docs/install_guides/installing-openvino-macos.md

```diff
@@ -112,9 +112,9 @@ Developing in Python:
    * [Start with PaddlePaddle models with OpenVINO](https://docs.openvino.ai/latest/notebooks/103-paddle-onnx-to-openvino-classification-with-output.html)
 
 Developing in C++:
-   * [Image Classification Async C++ Sample](../../samples/cpp/classification_sample_async/README.md)
-   * [Hello Classification C++ Sample](../../samples/cpp/hello_classification/README.md)
-   * [Hello Reshape SSD C++ Sample](../../samples/cpp/hello_reshape_ssd/README.md)    
+   * [Image Classification Async C++ Sample](https://docs.openvino.ai/latest/openvino_inference_engine_samples_classification_sample_async_README.html)
+   * [Hello Classification C++ Sample](https://docs.openvino.ai/latest/openvino_inference_engine_samples_hello_classification_README.html)
+   * [Hello Reshape SSD C++ Sample](https://docs.openvino.ai/latest/openvino_inference_engine_samples_hello_reshape_ssd_README.html)
```

Comment:
Our documentation is versioned. I believe we should use global links format here.  It's permanently points to the `latest`, whereas relative path routes to corresponding version of a page.
CC'ed: @andrew-zaytsev

Reply:
@slyubimt I agree but when I tried to use relative paths it didn't work on built documentation, you can check my attempt at previous commit https://github.com/openvinotoolkit/openvino/pull/9636/commits/76046a239541cc172869aa32b94c690dfb8e5adf

Reply:
@ntyukaev Can you help with this issue?